excellence-fee/excellence-fee.php

```diff
@@ -62,27 +52,62 @@ function fx_admin_notice_example_notice(){
 }
  
 function pluginprefix_install() {
-     	plugin_activation();
-    // clear the permalinks after the post type has been registered
+ 	plugin_activation();
+ 
     flush_rewrite_rules();
 }
 register_activation_hook( __FILE__, 'pluginprefix_install' );
 
 function pluginprefix_deactivation() {
-    // unregister the post type, so the rules are no longer in memory
-    // unregister_post_type( 'book' );
-    // clear the permalinks to remove our post type's rules from the database
+  
     flush_rewrite_rules();
 }
 register_deactivation_hook( __FILE__, 'pluginprefix_deactivation' );
 
+
+function excellence_fee_admininc() {
+
+    wp_enqueue_style( 'excellence_css', plugins_url( 'inc/style.css', __FILE__ ));
+    wp_enqueue_script( 'excellence_js', plugins_url( 'excellence-fee/inc/jquery-3.4.1.min.js', dirname(__FILE__)));
```

Comment:
@Shraddha1596 why jquery is included again when wordpress alreay have it, same mistake done on OMAR project, not learing from past mistakes, poor thing